### PR TITLE
welcome pack files upload

### PR DESCRIPTION
**Pull request by waqasgulzar**

### CODE REVIEW

File: src/app/shared/services/mediaLibrary.service.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  getUri(): string {
+    return `/api/medialibrary`;
+  }
+
+  getInstance(): Entity {
+    return new _model.MediaLibrary();
+  }
+
+  upload(formData: FormData) {
```
Comment (ghazanfarseyal) [inline src/app/shared/services/mediaLibrary.service.ts]: why you added this upload method here? As we already have it in file-service.
cant we use that ?

Comment (waqasgulzar) [reply src/app/shared/services/mediaLibrary.service.ts]: mediaLibrary and file-service have different API URI. thats why i have add it here.

if we can handle any other way, please advice.

File: src/app/modules/welcomePack/welcome-pack.component.ts [hunk 2]
```diff
@@ -30,54 +37,125 @@ export class WelcomePackComponent implements OnInit {
...
+      //id: [],
+      //mediaTypeId: [],
+      //documentTypeId: [],
+      fileName: [],
+      fileSize: []
+      //onDiskName: [],
+      //onDiskPath: [],
+      //createdBy: [],
+      //createdOn: [],
+      //isDeleted: false
```
Comment (ghazanfarseyal) [inline src/app/modules/welcomePack/welcome-pack.component.ts]: Remove commented code

File: src/app/modules/welcomePack/welcome-pack.component.ts [hunk 3]
```diff
@@ -30,54 +37,125 @@ export class WelcomePackComponent implements OnInit {
...
-    if (index >= 0) {
-      this.cclist.splice(index, 1);
-    }
+  // remove(cc): void {
+  //   const index = this.cclist.indexOf(cc);
+
+  //   if (index >= 0) {
+  //     this.cclist.splice(index, 1);
+  //   }
+  // }
```
Comment (ghazanfarseyal) [inline src/app/modules/welcomePack/welcome-pack.component.ts]: Remove commented code

File: src/app/modules/welcomePack/welcome-pack.component.ts [hunk 4]
```diff
@@ -30,54 +37,125 @@ export class WelcomePackComponent implements OnInit {
...
+  ) {}
 
   ngOnInit() {
     this.orgInfo = this.userInfo.currentUser;
-    this.types = [{ id: 1, title: 'Appointment Booking' }, {id:2, title:'Appointment Cancellation'}];
+    this.types = [
+      { id: 1, title: 'Appointment Booking' },
+      { id: 2, title: 'Appointment Cancellation' },
+      { id: 3, title: 'Customer Registration' }
+    ];
```
Comment (ghazanfarseyal) [inline src/app/modules/welcomePack/welcome-pack.component.ts]: dont do hard code, get it from api/lookup

**Comment by ghazanfarseyal** [comment]
ok

File: src/app/modules/welcomePack/welcome-pack.component.ts (hunk 4 above)
Comment (waqasgulzar) [reply src/app/modules/welcomePack/welcome-pack.component.ts]: done it.

File: src/app/modules/welcomePack/welcome-pack.component.ts (hunk 3 above)
Comment (waqasgulzar) [reply src/app/modules/welcomePack/welcome-pack.component.ts]: removed.

**Pull request merged**